projects/worker-animal-pictures/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "example-worker-animal-pictures",
+    "author": "Adobe Inc.",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "description": "Asset Compute example custom worker - Animal Pictures",
+    "repository": "adobe/example-worker-animal-pictures",
+    "dependencies": {
+        "@adobe/httptransfer": "^1.0.1",
+        "@adobe/asset-compute-sdk": "^1.0.2"
+    },
+    "devDependencies": {
+        "@adobe/aio-cli": "^3.4.1",
```

Comment:
I don't think `aio-cli` should be a devDependency? Unless I am missing something, users are expected to have it installed globally.

Reply:
For travis tests since script -> test run `aio asset-compute test-worker`

Reply:
Yeah, but then it should be installed inside the `.travis.yml`.

`aio-cli` is a _heavy_ beast, and having it as a `devDependency` slows down an `npm install` of the project dramatically and increases the `node_modules` size.

Reply:
ok Thanks!